packages/kotti-table/src/TableRow.js

```diff
@@ -145,20 +158,39 @@ export default {
 			$event.stopPropagation()
 			this[KT_STORE].commit('expandRow', row)
 		},
+		handleActionsClick($event) {
+			$event.stopPropagation()
```

Comment:
This should be  a `@click.stop`.

Reply:
same thing, jsx doesn't have the vue syntax sugare